model/wrapper.js

```diff
@@ -0,0 +1,44 @@
+var bcrypt = require('bcrypt')
+const saltRounds = 10
+
+function createUser(connection, uname, email, password, admin) {
+    bcrypt.hash(password, saltRounds, function (err, hash) {
+        var query = 'INSERT INTO users VALUES(' + email + ', ' + hash + ', ' + uname +  ', ' + admin + ')'
+        connection.query(query, function (err, rows, fields) {
+            if (err) {
+                throw "Failed to execute create user SQL"
+            } else {
+                return true
+            }
+        })
+    })
+}
+
+function userExists(connection, email) {
```

Comment:
What is the use-case for this wrapper statement?

Reply:
Check to make sure we're not making duplicate accounts